contrib/crazy_moles/Makefile

```diff
@@ -4,7 +4,7 @@ bin/moles: $(shell ../../bin/nitls -M src/moles_linux.nit) assets/images/drawing
 	mkdir -p bin
 	../../bin/nitc -o bin/moles src/moles_linux.nit
 
-android: android-icons $(shell ../../bin/nitls -M src/moles_android.nit) assets/images/drawing.png
+bin/moles.apk: android-icons $(shell ../../bin/nitls -M src/moles_android.nit) assets/images/drawing.png
```

Comment:
I suggest that an `android` rule still exists in project that can generate apk files. It is a simple convention for the human and allow mass invocation like #1652 

    android: bin/moles.apk